src/cgame/cg_animdelta.cpp

```diff
@@ -71,17 +71,7 @@ bool AnimDelta::ParseConfiguration(clientInfo_t* ci, const char* token2, const c
 bool AnimDelta::LoadData(clientInfo_t* ci)
 {
 	char newModelName[ MAX_QPATH ];
-	// special handling for human_(naked|light|medium)
-	if ( !Q_stricmp( ci->modelName, "human_naked"   ) ||
-		!Q_stricmp( ci->modelName, "human_light"   ) ||
-		!Q_stricmp( ci->modelName, "human_medium" ) )
-	{
-		Q_strncpyz( newModelName, "human_nobsuit_common", sizeof( newModelName ) );
-	}
-	else
-	{
-		Q_strncpyz( newModelName, ci->modelName, sizeof( newModelName ) );
-	}
```

Comment:
why is this required?

Reply:
This code was required when our human files were not following conventions. Now that the files are following convention, this custom loader can be dropped to rely on the generic one instead. In fact we should drop the custom loader to not look for the old files.

This is useless custom code being deleted. This was basically a workaround for the original files not following conventions.

Reply:
we can drop the copy too then

Reply:
Done.